esmvaltool/recipes/recipe_climwip_brunner2019_med.yml

```diff
@@ -276,13 +276,13 @@ diagnostics:
 
```

Comment:
For the other PRs, you added some settings here. Are they not needed for in this case?